docs/cristin.ttl

```diff
@@ -906,10 +906,75 @@ It is a self reference.""" ;
         entityDescription.reference.doi
     """ .
 
+:antall_sider_totalt
+    a               :cristinfield ;
+    rdfs:comment    "The total number of pages. Possibly null." ;
+    rdfs:range      rdf:nil ; #unknown range
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
+:publiseringsar
+    a               :cristinfield ;
+    rdfs:comment    "The year of publication" ;
+    rdfs:range      xsd:positiveInteger ;
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
+:hefte
+    a               :cristinfield ;
+    rdfs:comment    "The booklet containing the publication." ;
+    rdfs:range      rdf:nil ; #unknown range
```

Comment:
Run an SQL query in Athena to see what kind of values we get here.

Reply:
"hefte" oversettes som "issue" i en sånn tidsskriftspubliseringskontekst